types/textual/types.go

```diff
@@ -0,0 +1,20 @@
+package textual
+
+import (
+	"context"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+// ValueRenderer defines an interface to produce formatted output for all
+// protobuf types as well as parse a string into those protobuf types.
+//
+// The notion of "value renderer" is defined in ADR-050, and that ADR provides
+// a default spec for value renderers. However, we define an interface here, so
+// that optionally more value renderers could be built, for example, a
+// separate one for a different language.
+type ValueRenderer interface {
+	Format(context.Context, protoreflect.FieldDescriptor, protoreflect.Value) ([]string, error)
```

Comment:
Any opinions on this interface?

I thought of using `protoreflect.Message` first, but then looking at the first 4 lines of a tx:
```
Chain ID: simapp-1
Account number: 10
*Public Key: iR...==
Sequence: 2
```

there's some customization, so probably doing formatting field by field makes more sense.